.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: cisagov/setup-env-github-action@develop
+      - uses: cisagov/setup-env-github-action@improvement/support_tf_0.13
```

Comment:
This change is reverted in https://github.com/cisagov/cool-assessment-terraform/pull/136, but just wanted to mention it to make sure that it flows in to this PR.

Reply:
Yep.  Taken care of in commit 1110e86.